draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,265 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+                <xref target="RFC4566"/>, Section 5.8, and the bwtype and
+                bandwidth values stored.</t>
+            </list>
+          </t>
+          <t>Specific processing MUST also be applied for the following attribute lines:
+            <list style="symbols">
+              <t>Any "ice-lite", "ice-ufrag", "ice-passwd", or "ice-options" lines
+                MUST be parsed as specified in <xref target="RFC5245"/>, Sections
+                15.3-15.5, and, if valid, their values are stored.</t>
+              <t>Any "a=fingerprint" lines are parsed as specified in
+                <xref target="RFC4572"/>, Section 5, and, if valid, the set of fingerprint and algorithm values is stored.</t>
+              <t>If present, an "a=setup" line is parsed as specified in
+                <xref target="RFC4145"/>, Section 4, and, if valid, the value
+                is stored.</t>
+            </list>
+          </t>
+          <t>If the "m=" proto value indicates use of RTP, as decribed in
+            the <xref target="sec.profile-names"/> section above, the following
+            attribute lines MUST be processed:
+            <list style="symbols">
+              <t>The "m=" fmt value MUST be parsed as specified in <xref target="RFC4566"/>,
+              Section 5.14, and the individual values stored.</t>
+              <t>Any "a=rtpmap", a=fmtp", "a=ptime", or directional attributes
+              MUST be parsed as specified in <xref target="RFC4566"/>, Section 6,
+              and their values stored.</t>
+              <t>Any "a=ssrc" or "a=ssrc-group" attributes MUST be parsed as
+              specified in <xref target="RFC5576"/>, Sections 4.1-4.2, and their
+              values stored.</t>
+              <t>Any "a=extmap" attributes MUST be parsed as specified in
+                <xref target="RFC5285"/>, Section 5, and their values stored.</t>
+              <t>Any "a=rtcp-fb" attributes MUST be parsed as specified in
+                <xref target="RFC4585"/>, Section 4.2., and their values stored.</t>
+              <t>Any "a=rtcp-mux" attribute MUST be parsed as specified in
+                <xref target="RFC5761"/>, Section 5.1.1, and its presence or
+                absence flagged and stored.</t>
+            </list>
+          </t>
+          <t>Otherwise, if the "m=" proto value indicats use of SCTP, the following
+            attribute lines MUST be processed:
+            <list style="symbols">
+              <t>The "m=" fmt value MUST be parsed as specified in
+                <xref target="I-D.ietf-mmusic-sctp-sdp"/>, Section 4.3, and
+                the application protocol value stored.</t>
+              <t>An "a=sctp-port" attribute MUST be present, and it MUST be
+                parsed as specified in <xref target="I-D.ietf-mmusic-sctp-sdp"/>,
+                Section 5.2, and the value stored.</t>
+            </list>
+          </t>
```

Comment:
I am fine with that (aside; I think -opus and 2833 are already covered (via a=fmtp);  -bundle is already covered via a=group; 3605 isn't directly used by us (a=candidate used instead), and 6464/6465 are handled by extmap)

So that just leaves 5506, msid, trickle-ice, and vp8.